src/containers/ViewerPanel/test/selectors.test.ts

```diff
@@ -41,33 +42,7 @@ describe("ViewerPanel selectors", () => {
             expect(displayTimes).to.deep.equal({
                 roundedTime: 3,
                 roundedLastFrameTime: 15,
-                unitLabel: "s",
-            });
-        });
-
-        it("returns user-set non-second time unit", () => {
-            const mockState: State = {
-                ...initialState,
-                selection: {
-                    ...initialState.selection,
-                    time: 3,
-                },
-                metadata: {
-                    ...initialState.metadata,
-                    timeUnits: {
-                        magnitude: 1,
-                        name: "ns",
-                    },
-                    lastFrameTime: 15,
-                },
-            };
-
-            const displayTimes = getDisplayTimes(mockState);
-
-            expect(displayTimes).to.deep.equal({
-                roundedTime: 3,
-                roundedLastFrameTime: 15,
-                unitLabel: "ns",
+                roundedTimeStep: 0.1,
```

Comment:
did we lose a whole test here?

Reply:
Yes. `getDisplayTimes` no longer returns a unit label, so the purpose of this test "getDisplayTimes return user-set non-second time unit" is no longer applicable.